integration-tests/create-react-app/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "create-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1"
+  },
+  "optionalDependencies": {
+    "react-scripts": "1.0.11"
```

Comment:
Possibly Node 4 (and earlier) still represent a large chunk of the node market: https://semaphoreci.com/blog/2016/10/12/nodejs-versions-used-in-commercial-projects-2016-edition.html